qp_woltka/woltka.py

```diff
@@ -0,0 +1,322 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+from math import ceil
+from os import environ
+from os.path import join, basename, exists
+from glob import glob
+import re
+
+from qiita_client import ArtifactInfo
+
+# number of threads per job
+PPN = 8
+MEMORY = '64g'
+WALLTIME = '10:00:00'
+MERGE_MEMORY = '48g'
+MERGE_WALLTIME = '4:00:00'
+MAX_RUNNING = 8
+
+
+# this is a almost exact copy/paste from the original
+# qp_woltka/support_files/to-job-array.py
```

Comment:
What are the differences relative to that file?

Reply:
Is there any way to centralize the code if most of it is repeated between the two files?

Reply:
There are a few minimal (I think mainly parameters), I'm OK `removing qp_woltka/support_files/to-job-array.py` but I thought it will be convenient to keep it as reference (the reason is in support_files)

Reply:
Got it, well if it's mostly for archiving then git already has a copy of that file in this branch so it should be fine to delete it ✨